file name encryption/AES-SIV-512-B64URL.md

```diff
@@ -0,0 +1,184 @@
+# File Name Encryption using AES-SIV-512 + Base64-URL-Encoding
+
+## Directory Metadata
+
+Every directory requires certain metadata that affects the file name encryption of its direct children:
+
+* the seed used to derive keys
+* the directory ID
+
+This data is immutable and therefore linked with a directory eternally, surviving renames/moves. This data is stored in a file called `dir.uvf`, which is stored in two places:
+1. Within the parent dir (except for root), where it serves a link to the child dir
+2. In the child dir itself (allowing disaster recovery without the parent)
```

Comment:
```suggestion
2. In the child dir itself (allowing disaster recovery without the parent). 


> [!NOTE] Disaster recovery without the parent
> Imagine cleartext folder structure `a/b/c/` but sync fails and `a/b/` gets lost. With this information, `?/?/c/` and 
> all its children can still be recovered. The `dirId` required for name decryption would otherwise only be available 
> within the lost parent dir.
```